packages/jest/plugins/resolver.ts

```diff
@@ -37,7 +37,29 @@ function getCompilerSetup(rootDir: string) {
   const host = ts.createCompilerHost(compilerOptions, true);
   return { compilerOptions, host };
 }
-
+// https://jestjs.io/docs/upgrading-to-jest28#packagejson-exports
+const pkgNamesToTarget = new Set([
+  'nanoid',
+  'uuid',
+  'rxjs',
+  '@firebase/auth',
+  '@firebase/storage',
+  '@firebase/functions',
+  '@firebase/database',
+  '@firebase/auth-compat',
+  '@firebase/database-compat',
+  '@firebase/app-compat',
+  '@firebase/firestore',
+  '@firebase/firestore-compat',
+  '@firebase/messaging',
+  '@firebase/util',
+  ...(process.env.NX_JEST_PACKAGE_NAMES
```

Comment:
document env var usage 
should this be called something else? feels a bit ambiguous